app/controllers/v0/medical_copays_controller.rb

```diff
@@ -2,11 +2,11 @@
 
 module V0
   class MedicalCopaysController < ApplicationController
-    before_action(except: :send_new_statements_notifications) { authorize :medical_copays, :access? }
+    before_action(except: :send_statement_notifications) { authorize :medical_copays, :access? }
 
-    skip_before_action :verify_authenticity_token, only: [:send_new_statements_notifications]
-    skip_before_action :authenticate, only: [:send_new_statements_notifications]
-    skip_after_action :set_csrf_header, only: [:send_new_statements_notifications]
+    skip_before_action :verify_authenticity_token, only: [:send_statement_notifications]
```

Comment:
Endpoint is hidden behind feature flag now while in development. When launched this route will be accessed by another VA API using an API key and CIDR block restrictions rather than typical authenticated user tokens